jeeves/secrets.py

```diff
@@ -0,0 +1,174 @@
+import os
```

Comment:
We should drop this file and just apply the outputs from terraform. That's much easier.